modin/pandas/test/test_io.py

```diff
@@ -1160,7 +1124,7 @@ def wrapped_read_table(file, method):
 
     @pytest.mark.xfail(
         condition="config.getoption('--simulate-cloud').lower() != 'off'",
-        reason="The reason of tests fail in `cloud` mode is unknown for now - issue #2340",
+        reason="The reason of tests fail in `cloud` mode is unknown for now - issue #3264",
```

Comment:
same here and in other places mentioning 3264 I guess...

Reply:
I will return the old reference (https://github.com/modin-project/modin/issues/2340) to where I changed it. Where there was no reference before, I will specify https://github.com/modin-project/modin/issues/3264, since this problem contains all the errors in 'modin in the cloud'. This solution will be more convenient, because when we investigate the error, we will get into a problem where there is already a list of all the added errors.